src/styles/blocks/calendar.scss

```
@@ -0,0 +1,54 @@
+.calendar {
+  width: 750px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  gap: 1px;
+  padding: 10px;
+
+  @each $day, $colums-day in $days {
+    &--start-day_#{$day} > .calendar__day:first-child {
+      margin-left: $colums-day;
+    }
+  }
+
+  &__day {
+    @extend %font-style;
+    @extend %square-style;
+    cursor: pointer;
+    transition: ease-in-out 0.5s;
```

Always add transition property that should change smoothly.
```suggestion
    transition: background-color ease-in-out 0.5s, transform ease-in-out 0.5s;
```